README.md

```diff
@@ -236,6 +236,18 @@ As much as possible, colors should be directly imported from the style guide. Th
 @sk-yellow
 ```
 
+### Z-indexes
+
+To ensure correct relative elements stacking order z-index variables are provided:
+
+```less
+@z-index-header-overlay: 4;
```

Comment:
Is it worth leaving the actual values out of the readme, and only documenting their stacking order?